.packit.yaml

```diff
@@ -0,0 +1,25 @@
+---
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: initoverlayfs.spec
+upstream_tag_template: v{version}
```

Comment:
remove line 6 entirely. Given the version tagged doesn't begin with `v`, packit can just use the defaults.

That's currently causing the build failure as seen at https://copr.fedorainfracloud.org/coprs/packit/containers-initoverlayfs-28/build/6721833/